### ISSUE TITLE
Add Timezone (TZ) support to the image and enable ENV variable to configure

### ISSUE THREAD

**Issue by craigcurtin-dev** [LordMike/MBW.BlueRiiot2MQTT#69]
Currently looking at the logs to debug information from the console is frustrating at the timezone is GMT.

I have manually added it to my image - but would be nice for the master to reflect it and expose configuration through the ENV variables

Craig

**Comment by LordMike** [comment]
When you added it - was it just an environment variable?

If so, support is already there, and I can just add docs.
________________________________
From: craigcurtin-dev ***@***.***>
Sent: Thursday, May 5, 2022 5:57:54 PM
To: LordMike/MBW.BlueRiiot2MQTT ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [LordMike/MBW.BlueRiiot2MQTT] Add Timezone (TZ) support to the image and enable ENV variable to configure (Issue #69)


Currently looking at the logs to debug information from the console is frustrating at the timezone is GMT.

I have manually added it to my image - but would be nice for the master to reflect it and expose configuration through the ENV variables

Craig

**Comment by craigcurtin-dev** [comment]
Nope had to install tzdata and symlink - went through a few steps until I got it to work

Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________

**Comment by LordMike** [comment]
Ah – cool. Could you make a PR that I can accept – else I might just get around to it at some point 😊

**Comment by craigcurtin-dev** [comment]
Yep will do – will document the steps in it and send it over later today

**Comment by Joe00** [comment]
any news on this?
